origami.json

```diff
@@ -18,7 +18,8 @@
 		"required": [
 			"Map",
 			"fetch",
-			"Array.from"
+			"Array.from",
+			"Element.prototype.remove"
```

Comment:
👌

Reply:
FYI, the Page Kit team is uses the default Polyfill which `remove()` is apart of.